### PR TITLE
Fetch page contributors and last update date from git

### LINKED ISSUE #9: [Enhancement] Add information about the last page refresh
This will help you see how fresh the documentation is.
It would also be nice to add information - who updated the page.

**The example in the screenshot:**
Starting a project on GitBook - GitBook 2019-03-26 18-45-32

### PR DESCRIPTION
**Pull request by Mark-H**
### What does it do?

During render fetches contributors to a specific page and the last update date.

Schermafbeelding 2021-02-25 om 21 25 51

### Why is it needed?

#9 

### Related issue(s)/PR(s)

Fixes #9 

### To do

- [x] ~As a minimum, cache the history information so it doesn't have to do a git operation each page view~
- [x] OR: make it populate a table with the information when syncing so it's available from there
- [x] Move styles into the proper sass files (#82)
- [ ] ..

### CODE REVIEW

**Comment by digitalpenguin** [comment]
Nice!

**Comment by Mark-H** [comment]
It's deployed to docs.modx.org for testing.

File: public/template/src/scss/components/_contributors.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.c-history {
+    display: flex;
+    align-items: center;
+    margin: -.9375rem 0 1.875rem
+}
+.c-contributors {
+    display: flex;
+}
+
+.c-contributor {
```
Comment (Ibochkarev) [inline public/template/src/scss/components/_contributors.scss:10]:
```suggestion 
.c-contributor {
    border-radius: 15px;
    width: 30px;
    height: 30px;
    border: 2px solid #fff;
    margin-left: -10px;

    &:first-child {
        margin-left: 0;
    }
}`

**Review by Ibochkarev — CHANGES_REQUESTED** [review]

**Comment by Ibochkarev** [comment]
@Mark-H And if the avatar is clickable, then it will generally best of the best =)

**Comment by Mark-H** [comment]
Where should it link to?

**Comment by Ibochkarev** [comment]
> Where should it link to?

to your github account

**Comment by Mark-H** [comment]
Hmmmmm it doesn't look like I immediately have that information from the git log. Only the name (shown on hover) and the email address. 

Looks like to link contributions to a github user, it would need to talk to the github api, e.g. https://api.github.com/search/users?q=example@email.com

**Comment by Mark-H** [comment]
I'm not gonna do that now but perhaps we can do that in the future. Could even add like `/contributor/name-or-email-or-id` and give everyone that contributed their own contributor page, showing what they've added, maybe even feed it into a leaderboard of sorts..

**Comment by Ibochkarev** [comment]
these were thoughts out loud.  already looks very good.  thank

**Comment by Mark-H** [comment]
I do like the way you think. ;)

**Pull request merged**